### ISSUE TITLE
Problem with test?

### ISSUE THREAD

**Issue by jw2340** [learn-co-curriculum/javascript-rock-dodger#4]
It seems that the solution given on GitHub does not pass the test in IDE. 

In createRock function, 'top' and 'rock.style.top' are assigned the values below:
  var top = 0
  rock.style.top = top

I changed the code to below:
var top = rock.style.top

With this change, the code will pass the test. However, the variable top is now assigned a string value, which affects how the index.hml executes in a browser. It seems that whenever the variable 'top' is a number, the code cannot pass the test in IDE.

In short, the solution on GitHub executes correctly in a browser. But it does not pass the test in IDE.

**Comment by jw2340** [comment]
Error that shows up is below:

> java-script-rock-dodger@0.1.0 test /home/jinwang527-26164/code/labs/javascript-rock-dodger-bootcamp-prep-000  
> mocha -R mocha-multi --reporter-options nyan=-,json=.results.json                                                                                 

Killed_,------,  
npm ERR! Test failed.  See above for more details.  
/usr/local/rvm/gems/ruby-2.2.3/gems/learn-test-2.4.0/lib/learn_test/strategies/mocha.rb:49:in `results': undefined method`[]' for nil:NilClass (NoM
ethodError) ""  
        from /usr/local/rvm/gems/ruby-2.2.3/gems/learn-test-2.4.0/lib/learn_test/runner.rb:68:in `push_results'                                     
        from /usr/local/rvm/gems/ruby-2.2.3/gems/learn-test-2.4.0/lib/learn_test/runner.rb:19:in`run'  
        from /usr/local/rvm/gems/ruby-2.2.3/gems/learn-test-2.4.0/bin/learn-test:60:in `<top (required)>'                                           
        from /usr/local/rvm/gems/ruby-2.2.3/bin/learn-test:23:in`load'  
        from /usr/local/rvm/gems/ruby-2.2.3/bin/learn-test:23:in `<main>'                                                                           
        from /usr/local/rvm/gems/ruby-2.2.3/bin/ruby_executable_hooks:15:in`eval'  
        from /usr/local/rvm/gems/ruby-2.2.3/bin/ruby_executable_hooks:15:in `<main>'

**Comment by AnnJohn** [comment]
hi @jw2340 this looks like an environment specific issue. If you haven't done so already, could you use Ask a Question in Learn and someone from our team will help you out and troubleshoot the problem with you. Thanks!
